Documentation/network/kubernetes/compatibility-table.rst

```diff
@@ -148,7 +148,9 @@
 +-----------------+----------------+
 | v1.13.15        | 1.26.8         |
 +-----------------+----------------+
+| v1.13.16        | 1.26.8         |
++-----------------+----------------+
 | v1.13           | 1.26.8         |
 +-----------------+----------------+
-| latest / master | 1.26.7         |
+| latest / master |                |
```

Comment:
I'm not sure this empty value is expected :thinking:

Reply:
Looking at the current value (`1.26.7`), it seems this has been broken for a while. The reason seems to be the script in the v1.13 branch looks for branch `master` to update this line (which we renamed to `main` some time ago). I suspect the new symptom (empty value) may be due to the recent changes in how we call the script, now from the `release` repository, although I didn't investigate in details.

I'll fix manually, I won't bother trying to fix the script here since v1.13 will be EOL soon. However, we'll need to reconsider how we call check-crd-compat-table.sh in the future to be more resilient here.

Reply:
Change of plane - I quickly fixed the script, because 1) it's trivial, 2) I don't want the checks to fail in CI or if anyone try to validate the table locally, and 3) it avoids breaking the table update again next month if we haven't found a satisfactory long-term solution by then.